qemu/tests/dump_guest_core.py

```diff
@@ -95,7 +97,8 @@ def check_vmcore_file():
     logging.info("trigger core dump command: %s" % trigger_core_dump_command)
     process.run(trigger_core_dump_command)
     utils_misc.wait_for(lambda: os.path.exists(core_file), timeout=60)
-    check_core_file()
-    if dump_guest_core == 'on':
-        crash_cmd %= host_kernel_version
-        check_vmcore_file()
+    if arch != 'ppc64':
```

Comment:
Let's define a variable in configuration file so that it is easy to extend it when the user test it on the other archs, like:
.cfg:
```
ppc64:
    check_core_file = 'no'
```
.py:
```
if params.get('check_core_file', 'yes') == 'yes':
    check_core_file(arch)
    ...
```

Reply:
Ok, this is more reasonable, thank you!